src/plugins/vis_type_timelion/public/helpers/panel_utils.ts

```diff
@@ -127,61 +96,20 @@ function buildOptions(
   );
   const format = xaxisFormatterProvider(uiSettings)(interval);
 
-  const tickLetterWidth = 7;
-  const tickPadding = 45;
-
   const options = {
     xaxis: {
```

Comment:
In `timelion_vis_componet.tsx` I found only the usage of `options.xaxis.tickFormatter` from this object.
Why do we nee to save other params?
Could we just use `createTickFormat` instead of `buildOptions` ?

Reply:
Yep, I removed unused options and changed name of the function